lib/pipefy_message/providers/aws_client/sqs_broker.rb

```diff
@@ -65,6 +66,10 @@ def build_log_hash(arg)
         def handle_queue_name(queue_name)
           @is_staging ? "#{queue_name}-staging" : queue_name
         end
+
+        def handle_queue_protocol(queue_url)
+          ENV["ASYNC_APP_ENV"] == "development" ? queue_url : queue_url.sub(%r{^http://}, "https://")
```

Comment:
I suggest some other approach (like a configuration layer) to avoid reading variables directly from the environment and to make the code agnostic to which environment it is running on.

Reply:
Can you give an example of it?

ps: I had to implement that because the HTTPS protocol was not working in the local environment, and we made this 'protocol replace' in a previous PR for all environments.

Reply:
We can implement that in a new PR to refactor how we deal with configuration. A very simple and naive implementation would have a set of config files (prod, stag, dev etc) that are injected in classes.